armi/operators/snapshots.py

```diff
@@ -31,7 +31,10 @@ def createInterfaces(self):
         # disable fuel management and optimization
         # disable depletion because we don't want to change number densities for tn's >0 (or any)
         for toDisable in ["fuelHandler", "optimize", "depletion"]:
-            i = self.getInterface(toDisable)
+            i = self.getInterface(name=toDisable)
+            if i:
+                i.enabled(False)
+            i = self.getInterface(function=toDisable)
```

Comment:
Could this same thing be accomplished if you added `function=toDisable` onto the original call to `getInterface`? So it would be:
```python
i = self.getInterface(name=toDisable, function=toDisable)
```

Though I may have missed something as I do not know the context of the PR.

Reply:
nope, thats a good idea. thanks!